src/test/java/com/lambdaschool/foundation/controllers/exampleTest.java

```diff
@@ -0,0 +1,12 @@
+package com.lambdaschool.foundation.controllers;
```

Comment:
I would get rid of this file going into main. No issue keeping on a local branch, just don't want any unneeded classes.

Reply:
oops, I thought I had gotten rid of that. Now I certainly have.